spec/features/schools/index_spec.rb

```diff
@@ -3,8 +3,8 @@
 RSpec.describe 'As a visitor' do
   describe 'When I visit "/schools"' do
     it 'I see every school' do
-      school_1 = create(:school)
-      school_2 = create(:school)
+      school_1 = create(:school, name: Faker::Coffee.notes)
```

Comment:
I added Faker just to see what it was haha, this is just stubbing in a random coffee for the name 

![coffee](https://media.giphy.com/media/DrJm6F9poo4aA/giphy.gif)